crates/telio-model/src/features.rs

```diff
@@ -180,6 +183,10 @@ impl FeatureNurse {
     fn default_enable_nat_traversal_conn_data() -> bool {
         true
     }
+
+    fn default_state_duration_cap() -> u64 {
+        60 * 60 * 24
```

Comment:
Nit: It's always nice to have some comment next to constant like these to understand why the specific value was chosen.